packages/stellar-axelar-std/Cargo.toml

```
@@ -25,6 +25,7 @@ stellar-axelar-std-derive = { workspace = true }
 [features]
 testutils = ["soroban-sdk/testutils", "dep:hex"]
 derive = ["dep:stellar-axelar-std-derive"]
+alloc = []
```

```suggestion
alloc = ["soroban-sdk/alloc"]
```
does soroban-sdk need alloc feature enabled as well for this?

---

Yeah I had manually added the alloc feature for the soroban-sdk in `stellar-interchain-token-service/Cargo.toml` but this is probably a neater way, so I changed it to this now